Gemfile

```diff
@@ -20,17 +20,11 @@ gem 'uglifier', '>= 1.3.0'
 gem 'turbolinks', '~> 5'
 # Build JSON APIs with ease. Read more: https://github.com/rails/jbuilder
 gem 'jbuilder', '~> 2.5'
-# Use Redis adapter to run Action Cable in production
-# gem 'redis', '~> 3.0'
-# Use ActiveModel has_secure_password
-# gem 'bcrypt', '~> 3.1.7'
 
+gem 'rails-controller-testing'
 # ReactJS gem
 gem 'react-rails', '~> 1.0'
 
-# Use Capistrano for deployment
-# gem 'capistrano-rails', group: :development
```

Comment:
Not sure Capistrano is what we're going with for deploying. Do you have strong reasons for this? Maybe webpack can offer a better all in on build and deploy. Ok to leave it for now.